### ISSUE TITLE
Layover: Resolve screen-reader a11y gap

### ISSUE THREAD

**Issue by StephenRiosDev** [AlaskaAirlines/auro-form#1093]
### Description

Right now the new layover works perfectly from a functionality standpoint, but there is a small gap in a11y. Since we pass the focus to the button projected into the trigger, the screen reader does not read the expanded/collapsed state on initial focus, only on subsequent interactions.

Right now it looks like the best solution is to pass expanded/collapsed to the projected button via aria attributes, add a screen-reader element to the page, and update it according to the layover state to ensure that all browsers and screen-readers have consistent access to the layover state, tying it to the current behavior to ensure we don't perform screen-reader tasks for behaviors like tooltip.

This task is a pretty big lift.
